web/server/codechecker_server/api/report_server.py

```diff
@@ -42,7 +42,8 @@
     ReviewStatusRuleFilter, ReviewStatusRuleSortMode, \
     ReviewStatusRuleSortType, RunData, RunFilter, RunHistoryData, \
     RunReportCount, RunSortType, RunTagCount, \
-    SourceComponentData, SourceFileData, SortMode, SortType
+    SourceComponentData, SourceFileData, SortMode, SortType, \
+    ReviewStatus as API_ReviewStatus, DetectionStatus, CheckerInfo
```

Comment:
20 lines later (@✏️) the alphabetic order is kept in the change, but here it was violated.